### PR TITLE
feat(tinlake-ui): handle launching pools

### LINKED ISSUE #497: Introduce `Launch phase` pool status
### Background
FF2 is launching with known/existing investors only in the launch phase while ramping up the portfolio. This needs to be reflected in the UI, as DROP invesors won't be able to automatically onboard/invest at the beginning.

### Approach
We introduce an  pool stati to `upcoming` that defines how these pools are displayed to investors in the UI. Sth similar may be introduced for `Maker only` pools. Note that we may create a different section on the Dashboard for these pools once there are 2-3 of them.

### ACs
- Pill shown: 
    - No Change for now in pill behaviour, pool will be displayed as `Oversubscribed` until we have more than one pool and can also create an additional section at the boom of the Dashboard
- Pool order on Dashboard
    - If possible always keep at bottom of active poollist, just above `Upcoming` pools
- Pool Overview tab
    - Deactivate `Invest button`
    - Add tooltip: `This pool is launching with with exisiting investors while ramping up the portfolio. If you are interested in investing once the pool opens, please contact the issuer."`
- TIN/DROP component
    - Hide invest buttons
    - Show copy similar to what we currently have for TIN: `Interested in investing? This pool is launching with with exisiting investors while ramping up the portfolio. If you are interested in investing once the pool opens, please [contact the issuer](mailto: "IssuerEmail".`
    -

### PR DESCRIPTION
**Pull request by alessioscalici**
### Description

This pull request introduces handling of pools with "isLaunching" flag

Closes #497 

### Approvals

- [ ] Dev
- [ ] Product

### Screenshots

Screenshot 2021-11-09 at 15 36 55
Screenshot 2021-11-09 at 15 37 34
Screenshot 2021-11-09 at 15 38 03
### Impact

Overview page, Pool overview page (invest button), Investment page (TIN?DROP components)

### CODE REVIEW

**Review by onnovisser — APPROVED** [review]
👍

File: tinlake-ui/components/InvestAction/index.tsx [hunk 1]
```diff
@@ -85,6 +86,13 @@ const InvestAction: React.FC<Props> = (props) => {
         <Tooltip title="Upcoming pool" description="This upcoming pool is not open for investments yet">
           <Button primary label="Invest" disabled />
         </Tooltip>
+      ) : isLaunching && address ? (
+        <Tooltip
+          title="Launching pool"
+          description="This pool is launching with with exisiting investors while ramping up the portfolio. If you are interested in investing once the pool opens, please contact the issuer."
```
Comment (offerijns) [inline tinlake-ui/components/InvestAction/index.tsx:92]:
```suggestion
          description="This pool is launching with with existing investors while ramping up the portfolio. If you are interested in investing once the pool opens, please contact the issuer."
```

File: tinlake-ui/components/PoolList/index.tsx [hunk 2]
```diff
@@ -54,12 +54,34 @@ const toNumber = (value: BN | undefined, decimals: number) => {
 
 const isAlignedLeft = (c: Column) => c.header === 'Investment Capacity'
 
+// group pools to sort them (e.g. active pools first, then launching pools and then upcoming pools at the bottom)
```
Comment (offerijns) [inline tinlake-ui/components/PoolList/index.tsx:57]: Good idea 👍

File: tinlake-ui/containers/Investment/View/TrancheOverview.tsx [hunk 3]
```diff
@@ -234,6 +235,18 @@ const TrancheOverview: React.FC<Props> = (props: Props) => {
       </div>
     </Stack>
   )
+
+  const infoForLaunchingPool = (
+    <Info>
+      <Heading level="6" margin={{ bottom: 'xsmall' }}>
+        Interested in investing?
+      </Heading>
+      This pool is launching with exisiting investors while ramping up the portfolio. If you are interested in investing
```
Comment (offerijns) [inline tinlake-ui/containers/Investment/View/TrancheOverview.tsx:244]:
```suggestion
      This pool is launching with existing investors while ramping up the portfolio. If you are interested in investing
```

**Review by offerijns — APPROVED** [review]

**Review by offerijns — APPROVED** [review]

**Pull request merged**